bin/add-font-assets.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const fs = require('fs');
+
+const json = fs.readFileSync('package.json', 'utf8');
+
+const pack = JSON.parse(json);
```

Comment:
Not sure if it's better but I would do a `require('package.json')` to save me some typing.

Reply:
I tried it at first because it does look a lot cleaner but then node can't seem to find the file, it only works when I use the ```fs``` module. Maybe there is something wrong with my env?

Reply:
Think fs is relative to pwd and require to the file. Try `require(path.resolve(’package.json’))`

Reply:
Yes, require is relative to the file... I forgot about that!